habitat_baselines/rl/ppo/policy.py

```diff
@@ -89,14 +91,19 @@ def get_value(self, observations, rnn_hidden_states, prev_actions, masks):
         )
         return self.critic(features)
 
+    # @profiling_wrapper.RangeContext("Policy.evaluate_actions")
```

Comment:
is it possible to have a flag on the profiling_wrapper instead of commenting all these lines out?